samples/router/src/hooks/useCustomRouting.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from "react"
+import useSendbirdStateContext from "@sendbird/uikit-react/useSendbirdStateContext"
+import { GroupChannelHandler } from '@sendbird/chat/groupChannel'
+import { useNavigate } from 'react-router-dom'
+
+
+const simpleUUID = () => {
+  return Date.now().toString(36) + Math.random().toString(36).substr(2)
+}
+
+export function useCustomRouting({ currentChannel }) {
+  const store = useSendbirdStateContext()
+  const navigate = useNavigate()
+  const sb = store?.stores?.sdkStore?.sdk
+  const userId = store?.config?.userId
+  useEffect(() => {
+    const handlerId = simpleUUID()
+    if (sb?.groupChannel?.addGroupChannelHandler) {
+      sb.groupChannel.removeGroupChannelHandler(handlerId)
+      const handler = new GroupChannelHandler({
+        onUserBanned: (groupChannel, user) => {
+          if (groupChannel.url === currentChannel?.url && user?.userId === userId) {
```

Comment:
It seems you passed `channelUrl` from `routes/Channel` component 
https://github.com/sendbird/sendbird-uikit-react/pull/634/files#diff-cebbd30c8ec7f1be5cdc180fd2d6c315d8caa9d0830428bad7d6dcadc77cc2f2R12 but retrieving `url` again?

Reply:
ah yeah :D